benchmarks/benchmark.py

```diff
@@ -430,9 +430,11 @@ def run_benchmark():
 
 def modify_config_props_for_mms(pargs):
     shutil.copyfile(CONFIG_PROP_TEMPLATE, CONFIG_PROP)
-    if pargs.gpus:
-        with open(CONFIG_PROP, 'a') as f:
-            f.write('number_of_gpu={}'.format(pargs.gpus[0]))
+    with open(CONFIG_PROP, 'a') as f:
+        f.write('\nnumber_of_netty_threads=32')
```

Comment:
Is this not given by the min_workers in the scale call?

Reply:
Nope. It's separate from that.

Reply:
Netty threads is frontend.